src/reason_parser.mly

```diff
@@ -4095,12 +4228,16 @@ val_ident:
   | PERCENT                                     { "%" }
   | LESSGREATER                                 { "<>" }
   | LESSDOTDOTGREATER                           { "<..>" }
+  | SLASHGREATERLESS                            { "/><" }
```

Comment:
Where is it explained why we need this special token? What errors occur in the parser if we try to keep only the tokens `/>` and `<`.